torchrl/objectives/tqc.py

```diff
@@ -0,0 +1,278 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import warnings
+from dataclasses import dataclass
+
+import numpy as np
+import torch
+
+from tensordict import TensorDict, TensorDictBase
+from tensordict.nn import InteractionType, set_interaction_type, TensorDictModule
+from tensordict.utils import NestedKey
+from torchrl.data import CompositeSpec
+from torchrl.envs.utils import ExplorationType, set_exploration_type
+from torchrl.objectives.common import LossModule
+from torchrl.objectives.utils import ValueEstimators
+
+
+class TQCLoss(LossModule):
```

Comment:
the docstring should be added
It should include:
- a link to the arxiv of the paper or similar
- a short description of what this algo is about
- an example of how to create and run the module
- some notes regarding value estimation etc